### DISCUSSION TITLE
sync through ssh?

### DISCUSSION THREAD

**Discussion by tamis-laan** [GothenburgBitFactory/taskwarrior#2444] [Q&A]
Currently trying to set up taskwarrior server (taskd) on my raspberry pi which is a huge pain. It would be nicer and easier if I could sync clients through my server using ssh similar to git. Is such an option available??

**Comment by xeruf** [comment]
if you don't need to resolve any conflicts, you can simply use any syncing tool or even something as basic as rsync to sync the database and possibly the rc file ;)
